Libraries/Text/Text.windows.js

```diff
@@ -0,0 +1,323 @@
+/**
```

Comment:
Can we add a comment with the file this is derived from, e.g.:
https://github.com/facebook/react-native/blob/346309ec1c6a039117972f550b13213d9463b75b/Libraries/Text/Text.js

Reply:
It hasn't been done for others, but I'd like to start doing that :-)

Reply:
It hasn't been done for others, but I'd like to start doing that :-)

Reply:
Especially when this is updated as a one-off.